src/erlfmt_cli.erl

```diff
@@ -92,39 +96,45 @@ with_parsed(Name, Config) ->
             erlang:halt(127)
     end.
 
+set_difference(Files, Excludes) ->
+    sets:to_list(sets:subtract(sets:from_list(Files), sets:from_list(Excludes))).
```

Comment:
I think in this case list subtraction could also work `Files -- Excludes`. Is there a specific reason for going with the set approach?

Reply:
list difference would only work if the lists contain no duplicates.
```erlang
[{files, [a.erl, a.erl]}, {files, [a.erl]}, {exclude_files, [a.erl]}]
```